### DISCUSSION TITLE
Recursos informáticos como Anexo

### DISCUSSION THREAD

**Discussion by solmos** [PaulaMorBla14/TFG-Baloncesto#6] [Ideas]
Creo que esta sección quedaría mejor como un Anexo al final del documento. También habría que añadir el lenguaje de programación R y la herramienta Rmarkdown y su relación con la investigación reproducible a esta sección.

https://github.com/PaulaMorBla14/TFG-Baloncesto/blob/e706cf14d27adceebf21ec3934363fb9cce801ed/Project_v1.Rmd#L27

**Comment by PaulaMorBla14** [comment]
mmmh vale! más que nada lo añadí porque como una parte de mi trabajo es el aprender y saber como funciona GitHub, pues pensé que debía añadirlo aquí para que dejara constancia. Viernes lo comentamos con Jose sino, por que fue él que propuso que esto fuera una parte de mi trabajo.
